std/range/primitives.d

```diff
@@ -1535,43 +1488,17 @@ static assert(isForwardRange!(typeof(r[1 .. 2])));
 static assert(hasLength!(typeof(r[1 .. 2])));
 ----
```

Comment:
Expand the diff her. The example in the function comment uses a copy/pasted version of the old declaration.